src/kernel/task/Mybuild

```diff
@@ -59,6 +59,8 @@ module task_resource {
 
 	@NoRuntime depends embox.compat.libc.assert
 	@NoRuntime depends embox.util.Array
+	//FIXME
+	@NoRuntime depends embox.kernel.task.resource.atexit
```

Comment:
Is it because it's required by limits.h ?

Reply:
Yes. I've no idea, how to do this more pretty